mlc_llm/relax_model/llama.py

```diff
@@ -1362,12 +1367,13 @@ def get_model(args, hf_config):
         max_window_size=config.max_sequence_length,
         stop_tokens=[2],
         add_prefix_space=False,
+        prefill_chunk_size=prefill_chunk_size,
```

Comment:
With the new `UpdateConfigFromMetadata()` in `llm_chat.cc`, we might want the `prefill_chunk_size` in `metadata_func` and `mlc-chat-config.json` to match, since we read from both places.

Currently, for non-Mistral models, if we do not specify `prefill_chunk_size` as an argument when building the model, we would record `prefill_chunk_size` as `config.max_sequence_length` in `metadata_func` and `-1` in `mlc-chat-config.json`. To unify them, we have two options: 

1. Record `prefill_chunk_size` to be `-1` in `metadata_func` (this way in `llm_chat.cc`, `prefill_chunk_size_` would be `-1` after `UpdateConfigFromMetadata()`, the default value)
2. Record `prefill_chunk_size` to be `config.max_sequence_length` in `mlc-chat-config.json`. This way, we **always** perform chunking, just like in `mistral.py`. 
    - i.e. we change from `prefill_chunk_size = config.max_sequence_length` to `args.prefill_chunk_size = config.max_sequence_length`.

Reply:
Fixed, thanks @CharlieFRuan